<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局背景色和字体颜色 */
QWidget {
    background-color: #F5F5F7; /* 浅灰色背景，类似苹果风格 */
    color: #333333; /* 深灰色字体 */
    font-family: &quot;Microsoft YaHei&quot;, &quot;微软雅黑&quot;, &quot;PingFang SC&quot;, sans-serif; /* 设置更美观的字体 */
}

/* 侧边栏按钮样式 */
#sidebarWidget QPushButton {
    border: none; /* 移除边框 */
    padding: 10px;
    text-align: left; /* 文字左对齐 */
    border-radius: 8px; /* 圆角 */
    font-size: 14px;
}

/* 侧边栏按钮 - 鼠标悬停效果 */
#sidebarWidget QPushButton:hover {
    background-color: #E8E8ED;
}

/* 侧边栏按钮 - 当前选中效果 (需要代码配合) */
#sidebarWidget QPushButton:checked {
    background-color: #FFFFFF;
    font-weight: bold;
}

/* 麦克风按钮样式 */
#micButton {
    min-height: 50px; /* 增大最小高度 */
    max-width: 200px; /* 限制最大宽度 */
    border: 1px solid #DCDCE1;
    background-color: #FFFFFF;
    border-radius: 12px;
    font-size: 16px;
}

/* 麦克风按钮 - 鼠标悬停效果 */
#micButton:hover {
    border-color: #C7C7CC;
}

/* 麦克风按钮 - 按下效果 */
#micButton:pressed {
    background-color: #F5F5F7;
}

/* 欢迎语标签样式 */
#welcomeLabel {
    font-size: 18px;
    color: #000000;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="sidebarWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>570</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="mainMenuButton">
          <property name="text">
           <string>Main</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::GoHome"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingsMenuButton">
          <property name="text">
           <string>Setting</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="mainStackedWidget">
       <property name="minimumSize">
        <size>
         <width>660</width>
         <height>580</height>
        </size>
       </property>
       <widget class="QWidget" name="mainPage">
        <widget class="QLabel" name="welcomeLabel">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>491</width>
           <height>151</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>欢迎使用AI助手，您可以按空格键或者点击麦克风按钮开始使用</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="micButton">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>380</y>
           <width>202</width>
           <height>52</height>
          </rect>
         </property>
         <property name="text">
          <string>开始聆听</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::AudioInputMicrophone"/>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleButton">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>11</y>
           <width>98</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="settingsPage">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>100</y>
           <width>271</width>
           <height>171</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>系统设置页面</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
